test/e2e/pkg/testnet.go

```diff
@@ -108,6 +111,8 @@ type Testnet struct {
 	DefaultZone                                          string
 	PbtsEnableHeight                                     int64
 	PbtsUpdateHeight                                     int64
+	LanePriorities                                       []uint32
```

Comment:
What is the difference between the new fields `LoadLaneWeights`, and `LanePriorities`... I'd say we only need one of them?

Reply:
`LanePriorities` stores the lanes as defined by the app when the testnet is initialised, so we don't fetch them every time we generate a tx.

Reply:
Ah OK, I think I finally got it:

* `LanePriorities` get the value from `app.LaneDefinitions()` which is hard-coded (so always the same). Always three lanes: 9, 4, 1
* `LoadLaneWeights` is configurable via the manifest and defines, for each of the three lanes, how often the load generator is gonna generate TXs for that lane

If this is not accurate, please let me know.

Reply:
That's correct. I added a few comments to clarify.